cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -62,3 +62,5 @@ alpha:
 
 metrics:
   enabled: false
+  monitor:
+    enabled: true
```

Comment:
I think we should probably make this `false` by default.

Reply:
I forgot to disable this after testing.